### ISSUE TITLE
[Bug] ptxas /tmp/tmpxft_000879fb_00000000-7_gemm_s4_f16.compute_80.ptx, line 208; fatal   : Parsing error near ':': syntax error

### ISSUE THREAD

**Issue by pangr** [InternLM/lmdeploy#279]
### Checklist

- [ *] 1. I have searched related issues but cannot get the expected help.
- [ *] 2. The bug has not been fixed in the latest version.

### Describe the bug

**when compiling on V100**

```
[ 35%] Building CUDA object src/turbomind/kernels/gemm_s_f16/CMakeFiles/gemm_s4_f16.dir/gemm_s4_f16.cu.o
nvcc warning : incompatible redefinition for option 'optimize', the last value of this option was used
ptxas info    : 52 bytes gmem
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi8ELi256ELi32ELi8ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi8ELi256ELi32ELi8ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 26 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi16ELi256ELi32ELi16ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi16ELi256ELi32ELi16ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi2ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi2ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 27 registers, 4096 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 27 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi16ELi256ELi32ELi16ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi16ELi256ELi32ELi16ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi32ELi128ELi32ELi32ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi32ELi128ELi32ELi32ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 48 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi32ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi32ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 1536 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi64ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi64ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 72 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi96ELi64ELi32ELi96ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi96ELi64ELi32ELi96ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 126 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 1536 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi64ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi64ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 164 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi8ELi256ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi8ELi256ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 27 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi16ELi256ELi32ELi16ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi16ELi256ELi32ELi16ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi32ELi64ELi64ELi32ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi32ELi64ELi64ELi32ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 79 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi32ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi32ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi64ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi64ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 168 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_70'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 32 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : 52 bytes gmem
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi8ELi256ELi32ELi8ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi8ELi256ELi32ELi8ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 26 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi16ELi256ELi32ELi16ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi64ELi16ELi256ELi32ELi16ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 42 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi2ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi2ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 26 registers, 4096 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi8ELi512ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 26 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi16ELi256ELi32ELi16ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi16ELi256ELi32ELi16ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 43 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi32ELi128ELi32ELi32ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi32ELi128ELi32ELi32ELi64ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 64 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi32ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi32ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 40 registers, 1536 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi64ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi64ELi64ELi32ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 72 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi96ELi64ELi32ELi96ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi96ELi64ELi32ELi96ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 126 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 43 registers, 1536 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi64ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi128ELi128ELi64ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 164 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi8ELi256ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi8ELi256ELi32ELi8ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 25 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi16ELi256ELi32ELi16ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi16ELi256ELi32ELi16ELi128ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 40 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi32ELi64ELi64ELi32ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi32ELi64ELi64ELi32ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 79 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi32ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi32ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 64 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi64ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi64ELi64ELi64ELi64ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 168 registers, 6144 bytes smem, 400 bytes cmem[0]
ptxas info    : Compiling entry function '_ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii' for 'sm_75'
ptxas info    : Function properties for _ZN9turbomind14gemm_s4_f16_nnINS_4GemmILi256ELi128ELi32ELi32ELi128ELi32ELi3ELi128ENS_9OutputOpsIJNS_3ops8IdentityENS3_14SiluActivationEEEEEEEEvP6__halfPKjPKS8_PK7__half2iiii
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 43 registers, 3072 bytes smem, 400 bytes cmem[0]
ptxas /tmp/tmpxft_000879fb_00000000-7_gemm_s4_f16.compute_80.ptx, line 208; fatal   : Parsing error near ':': syntax error
ptxas fatal   : Ptx assembly aborted due to errors
make[2]: *** [src/turbomind/kernels/gemm_s_f16/CMakeFiles/gemm_s4_f16.dir/build.make:77: src/turbomind/kernels/gemm_s_f16/CMakeFiles/gemm_s4_f16.dir/gemm_s4_f16.cu.o] Error 255
make[1]: *** [CMakeFiles/Makefile2:2365: src/turbomind/kernels/gemm_s_f16/CMakeFiles/gemm_s4_f16.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

### Reproduction

make

**Comment by lzhangzz** [comment]
What's the version of your CUDA toolkit?

**Comment by pangr** [comment]
> What's the version of your CUDA toolkit?

CUDA 11.3

**Comment by lzhangzz** [comment]
@pangr 

Can you please give #280 a try to see if it helps?

**Comment by pangr** [comment]
> @pangr
> 
> Can you please give #280 a try to see if it helps?

okok,Thanks for the reply
